packages/app/src/feature-flags/constants/feature.enum.ts

```diff
@@ -4,16 +4,36 @@
  */
 
 export enum Feature {
-  LoggerEnabled = "logger-enabled",
-  LogsScrubbed = "logs-scrubbed",
-  DeveloperModeHidden = "developer-mode-hidden",
-  DisabledOnProduction = "disabled-on-production",
-  McPrereleaseAvailable = "mc-prerelease-available",
-  MessagesHidden = "messages-hidden",
-  DevelopOnly = "development-only",
-  ProductionAndAlpha = "production-and-alpha",
-  MessagesSearch = "messages-search",
   ProductionReleaseOnly = "production-release-only",
   TestProductionReleaseOnly = "test-production-release-only",
   AllReleaseListAvailable = "all-release-list-available",
+
+  LoggerEnabled = "logger-enabled",
+  LogsScrubbingEnabled = "logs-scrubbing-enabled",
+  DeveloperModeEnabled = "developer-mode-enabled",
+  FilesManagerEnabled = "files-manager-enabled",
+  MessagesSearchEnabled = "messages-search-enabled",
+  MessagesThreadDeleteEnabled = "messages-thread-delete-enabled",
+  MessagesThreadAttachContactEnabled = "messages-thread-attach-contact-enabled",
```

Comment:
The value is duplicated for `MessagesThreadAttachContactEnabled` and `MessagesThreadAttachTemplateEnabled`.